apps/server/src/modules/tool/controller/dto/response/context-external-tool.response.ts

```diff
@@ -7,22 +7,22 @@ export class ContextExternalToolResponse {
 	id: string;
 
 	@ApiProperty()
-	schoolToolId!: string;
+	schoolToolId: string;
 
 	@ApiProperty()
-	contextId!: string;
+	contextId: string;
 
 	@ApiProperty({ enum: ToolContextType })
-	contextType!: ToolContextType;
+	contextType: ToolContextType;
 
-	@ApiProperty()
-	contextToolName?: string;
+	@ApiPropertyOptional()
+	contextToolName: string;
```

Comment:
you removed the optional from the variable itself

Reply:
thanks :)